src/Android/Accessibility/AccessibilityHelpers.cs

```diff
@@ -129,16 +129,205 @@ public static class AccessibilityHelpers
             "com.ss.squarehome2",
             "com.treydev.pns"
         };
-        
-        // Be sure to keep these entries sorted alphabetically
+
+        // Be sure to keep these sections sorted alphabetically
         public static Dictionary<string, KnownUsernameField> KnownUsernameFields => new List<KnownUsernameField>
         {
-            new KnownUsernameField("accounts.google.com",   new (string, string)[] { ("ServiceLogin", "Email") }),
-            new KnownUsernameField("amazon.com",            new (string, string)[] { ("signin", "ap_email_login") }),
-            new KnownUsernameField("github.com",            new (string, string)[] { ("", "user[login]-footer") }),
-            new KnownUsernameField("paypal.com",            new (string, string)[] { ("signin", "email") }),
-            new KnownUsernameField("signin.aws.amazon.com", new (string, string)[] { ("signin", "resolving_input") }),
-            new KnownUsernameField("signin.ebay.com",       new (string, string)[] { ("eBayISAPI.dll", "userid") }),
+            /**************************************************************************************
+             * SECTION A ——— World-renowned web sites/applications
+             *************************************************************************************/
+
+
```

Comment:
As a general note please limit vertical white-space to 1 blank line per "block".

Reply:
OK, I will look to adapt this while maintaining good readability of the whole.

Reply:
Done. Is it better now? :relaxed:

Reply:
Further reduced spacing. :thumbsup:

Reply:
yes, thank you, would just like to avoid double blank lines (there are still quite a few in there), readability is great still where you've reduced so far due your use of comment blocks.

Reply:
Done. And, indeed, readability is still good after this change. Great! Is it OK for you now?

Reply:
I will mark this as resolved, therefore (since the PR is now merged). :thumbsup: